### PR TITLE
Issue-16603 Changed Kiev to Kyiv

### LINKED ISSUE #16603: Change the spelling of the Kiev to Kyiv in the settings timezone picker
The timezone picker shows a russian version of spelling "Kiev" and should be "Kyiv" to be more like Ukrainian spelling of Київ.

Screenshot of the Admin UI's `/settings/general` screen:
Image

A [similar issue in momenjs](https://github.com/moment/moment/issues/6008). We need to investigate first, which library the Admin UI is using to fetch the timezone data into the dropdown.

### PR DESCRIPTION
**Pull request by jonock**
This should contribute to fix [Issue-16603](https://github.com/TryGhost/Ghost/issues/16603)

---

### <samp>🤖 Generated by Copilot at 5467b82</samp>

Updated the spelling of 'Europe/Kiev' to 'Europe/Kyiv' in the `timezones.js` fixture file. This is part of a pull request that fixes timezone names in the Ghost admin interface.

### CODE REVIEW

**Comment by naz** [comment]
Hey @jonock 👋 Thanks for the contribution! This is a right change to do. There's one more piece missing - bumping the dependendy `@tryghost/timezone-data` where the Admin UI app [pulls the displayed names](https://github.com/TryGhost/Ghost/blob/d4e9c23f84d1744555dbfbfc2c28ccf37e054edb/ghost/admin/app/instance-initializers/config.js#L7) from. I have just [released](https://github.com/TryGhost/SDK/commit/8e1168f2b5e4854791cb3e3d9266f96c7db27912) an updated version of the timezone-data package, so all is left to do here is upgrade to the `@tryghost/timezone-data@0.3.0` and the fix is done 🎉  Can you please bump the version and I'll make sure to merge the change asap. Thanks!

File: ghost/admin/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@tryghost/nql": "0.11.0",
     "@tryghost/nql-lang": "0.5.0",
     "@tryghost/string": "0.2.4",
-    "@tryghost/timezone-data": "0.2.76",
+    "@tryghost/timezone-data": "0.3.0",
```
[the remark this answers is not in the document]
Comment (jonock) [reply ghost/admin/package.json:56]: Sorry for the mess with the commits...

[the remark this answers is not in the document]
Comment (naz) [reply ghost/admin/package.json:56]: If you run `yarn` command from the core/admin folder the yarn.lock file will update automatically and show up in the changeset to commit. No worries about the commits, it's all good as long as we clean up before merging into main branch 😄

[the remark this answers is not in the document]
Comment (jonock) [reply ghost/admin/package.json:56]: Thank you @naz!
I updated the yarn.lock file accordingly

[the remark this answers is not in the document]
Comment (naz) [reply ghost/admin/package.json:56]: For example, this is how much change I see locally when the timezone-data package is bumped:
image

[the remark this answers is not in the document]
Comment (jonock) [reply ghost/admin/package.json:56]: @naz ok it looks cleaner now but the casper changes are still there :(

**Pull request closed without merging**